src/main/java/bio/terra/common/exception/InternalServerErrorException.java

```diff
@@ -0,0 +1,32 @@
+package bio.terra.common.exception;
+
+import org.springframework.http.HttpStatus;
+
+import java.util.List;
+
+/**
+ * This exception maps to HttpStatus.INTERNAL_SERVER_ERROR in the GlobalExceptionHandler.
+ */
+public abstract class InternalServerErrorException extends ErrorReportException {
```

Comment:
worth a comment on why this is an `abstract class` but none of the other subclasses of ErrorReportException are.